server/routes.go

```diff
@@ -0,0 +1,13 @@
+package server
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+func setupRoutes(router *gin.Engine) {
+	// It is good practice to version your API from the start
+	v1 := router.Group("/api/v0.1")
```

Comment:
we are not having any versioning of APIs

Reply:
In future we might change these right, not right now but still i think it is still good to have